bats/fb-finish.bats

```diff
@@ -1,12 +0,0 @@
-#!/usr/bin/env bats
-# vim: ft=sh:sw=2:et
-
-set -o pipefail
-
-load os_helper
-load foreman_helper
-
-@test "collect important logs" {
-  tail -n100 /var/log/{apache2,httpd}/*_log /var/log/foreman{-proxy,}/*log /var/log/messages > /root/last_logs || true
-  foreman-debug -q -d /root/foreman-debug || true
```

Comment:
sosreport on Debian doesn't call foreman-debug so we would lose that info. I don't mind seeing this go in the Ansible role, but that info is useful to have.

Reply:
Do we collect this information today? I couldn't find any references to that path in Forklift or foreman-infra.

Reply:
I don't think this is collected (as in moved anywhere non-ephemeral) today.

I vote "nuke it here" and let's add a separate "foreman_debug" role that can get loaded on Debian?

Reply:
Sounds good.